recipes/lemon/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "lemon" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "71b7c725f4c0b4a8ccb92eb87b208701586cf7a96156ebd821ca3ed855bad3c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ version  }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+   # NOTE: this disables the installation of the glpk dlls
+   # from the lemon build system. An issue has been raised upstream
+   # for this:
+   # http://lemon.cs.elte.hu/pipermail/lemon-devel/2017-May/000392.html
+    - glpk_win.diff    # [win]
+
+build:
+  number: 0
+  features:
+    - vc9           # [win and py27]
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python        # [win]
+    - toolchain
+    - cmake
+    - glpk
+    - vc 9          # [win and py27]
+    - vc 14         # [win and py>=35]
+    - coincbc       # [unix]
+    - doxygen
+    - graphviz
+    - ghostscript   # [unix]
+  run:
+    - glpk
+    - vc 9          # [win and py27]
+    - vc 14         # [win and py>=35]
```

Comment:
Would recommend doing something like `# [win and (py35 or py36)]` just to avoid potential issues with Python 3.7 when it comes out. Though this is not a blocker just a suggestion.